comms/storage/storageserver/weather-map/src/App.tsx

```diff
@@ -2,23 +2,28 @@ import { QueryClient, QueryClientProvider } from 'react-query'
 import { createBrowserRouter, RouterProvider } from 'react-router-dom'
 
 import Home from './routes/Home'
+import Search from './routes/Search'
 import Shard from './routes/Shard'
 import Wrapper from './routes/Wrapper'
 
 const queryClient = new QueryClient()
-const baseURL = '/storage/weather'
+const baseURL = '/storage'
 
 export default function App() {
   const router = createBrowserRouter(
     [
       {
-        path: `/`,
+        path: '/',
         element: <Wrapper />,
         children: [
           {
-            path: `/`,
+            path: '/',
             element: <Home />,
           },
+          {
+            path: '/search/:query',
+            element: <Search />,
+          },
```

Comment:
`/storage/search` endpoint to programmatically search without having to duplicate the logic of parsing out a search query into either shard, file name, or job ID